src/Kirel.Identity.Server.API/appsettings.json

```diff
@@ -5,6 +5,9 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "ControllerState": {
```

Comment:
I think, we should rename this property name, and use something like:
"RegistrationEnable":  true/false, without info about controllers and other details of the implementations.